### PR TITLE
Add Remote Start button for Washer Devices

### PR DESCRIPTION
**Pull request by ollo69**
- Add Remote Start button (tested on ThinqV1 devices)
- Change Power Off switch to Power switch (breaking change)
- Use Power switch to wake-up device from stand-by (tested on ThinqV1 devices)
- Fix issue on ThinqV1 devices that could cause loss of connection

### CODE REVIEW

**Comment by purcell-lab** [comment]
Thanks, these controls are a welcome addition, but they appear to be unavailable for me :-(
Screenshot_20230209-072014

**Comment by purcell-lab** [comment]
Screenshot_20230209-072118

**Comment by ollo69** [comment]
@purcell-lab,

this is a closed PR, please open a new issue.
In any case as explained in release note: 
- `power-off` switch can be removed because is replaced by `power` switch
- `power` switch is available only when device is on or in stand-by mode (not when off, not possible to remotely turn on the device)
- `remote start` button is available only when remote start mode is enabled from device panel.

Basically same options also available in ThinQ app

**Comment by purcell-lab** [comment]
Thanks user error on my behalf.  All good here and this will work nicely with my excess solar integration.

Load washer, switch to remote start mode, then HA can switch on later in the day when energy is cheap and green.

Screenshot_20230209-075854
